great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -980,7 +983,7 @@ def get_validator_with_expectation_suite(
 
 
 def get_or_create_expectation_suite(
-    data_context: AbstractDataContext,
+    data_context: Optional[AbstractDataContext],
```

Comment:
@Kilo59 has convinced me that if no default is set, `Union[AbstractDataContext, None]` should be used. This makes it clear to the developer that this is not in fact `Optional`, but is a required parameter that accepts `None`.

Reply:
@NathanFarmer I am fine with this as long we make a wholesale change to the codebase -- and should get agreement from others as well.  Thanks.

Reply:
@alexsherstinsky I will create a Jira ticket for this change